steamfriends.cpp

```diff
@@ -92,7 +94,7 @@ bool SteamFriends_Invoke(void *SteamFriendsContext, echandle MethodDictionaryHan
         RetVal = IDictionary_GetInt32ByKey(MethodDictionaryHandle, "index", &Value32);
         if (RetVal == true)
             SteamFriends_GetFriendByIndex(Value32, (uint64_t *)&Value64);
-        String_Print(Value64String, Element_Count(Value64String), "%lld", Value64);
+        String_Print(Value64String, Element_Count(Value64String), "%" PRIu64, (uint64_t)Value64);
```

Comment:
Isn't this supposed to go in the quote?

Reply:
I like it. Also always wanted to do it. It enabled compiler warnings if wrong arguments are used.

Reply:
Good point but still makes it harder to read.